libs/util/http.client.js

```diff
@@ -103,41 +107,34 @@ function mergeConfig(config) {
         }
 
         if (config.retry) {
-            interval = parseInt(config.retry && config.retry.interval, 10);
+            interval = parseInt(config.retry.interval, 10);
             if (!isNaN(interval) && interval > 0) {
                 cfg.retry.interval = interval;
             }
-            maxRetries = parseInt(config.retry && config.retry.max_retries, 10);
+            maxRetries = parseInt(config.retry.max_retries, 10);
             if (!isNaN(maxRetries) && maxRetries >= 0) {
                 cfg.retry.max_retries = maxRetries;
             }
+            if (config.retry.statusCodes) {
+                cfg.retry.statusCodes = config.retry.statusCodes
```

Comment:
Lets document this new option in the readme please.

Reply:
Done. But while documenting I felt that it was so wrong to use `max_retries` while all the other settings use camel case names that I added another commit deprecating `max_retries` in favor of `maxRetries`.